hera_mc/part_connect.py

```diff
@@ -70,15 +70,14 @@ def part(self, **kwargs):
             setattr(self, key, value)
 
 
-def stop_existing_parts(session, p, hpnr_list, at_date, actually_do_it):
+def stop_existing_parts(session, hpnr_list, at_date, actually_do_it):
     """
     This adds stop times to the previous parts.
 
     Parameters:
     ------------
     session:  db session to use
-    p:  part object
-    hpnr_list:  list containing hpn and revision
+    hpnr_list:  list containing hpn
```

Comment:
It seems as if the old docstring for `hpnr_list` was more precise. Is there a terminology change here?